common/src/main/java/org/wso2/testgrid/common/TestScenario.java

```diff
@@ -75,6 +77,14 @@
     @Element(description = "holds the test engine type (i.e. JMETER, TESTNG)")
     private TestEngine testEngine;
 
+    @Column(name = PRE_SCRIPT_STATUS_COLUMN)
```

Comment:
Column name here should be `pre_script` as per the database naming conventions. However public static columns names should be kept for the ease of the developer to create queries.

Reply:
Donw